CMakeLists.txt

```diff
@@ -69,7 +69,10 @@ endif(LLVM_SPIRV_INCLUDE_TESTS)
 install(
   FILES
     ${LLVM_SPIRV_INCLUDE_DIRS}/LLVMSPIRVLib.h
+    ${LLVM_SPIRV_INCLUDE_DIRS}/LLVMSPIRVLibC.h
```

Comment:
Yeah, that's probably better. I wanted to avoid invasive changes, like changing the header folder structure, as I don't know what kind of stability guarantees are offered (esp. to keep these changes backportable).